etcdutl/snapshot/util.go

```diff
@@ -16,16 +16,20 @@ package snapshot
 
 import (
 	"encoding/binary"
+	"fmt"
 )
 
 type revision struct {
 	main int64
 	sub  int64
 }
 
-func bytesToRev(bytes []byte) revision {
+func bytesToRev(bytes []byte) (revision, error) {
```

Comment:
is it worth updating `mvcc.bytesToRev` with the same check? I don't know why these functions are separate. Is it possible to use same function in both places?

Reply:
I think this function is copied to several places because it is not exported from mvcc package and it's small. I can export the one in mvcc and remove others if you're okay with the change. It's better to do in in another PR though. Let me know.

Reply:
maybe we can add comment like, "must be kept in sync with  ../../server/storage/mvcc/xxx`, if we don't export the function from mvcc.